Cargo.toml

```diff
@@ -38,6 +38,9 @@ actix-web = "4"
 [build-dependencies]
 prost-build = { version = "0.9.0", optional = true }
 
+[[example]]
```

Comment:
Cross-referencing question on whether this is needed: https://github.com/prometheus/client_rust/pull/94#discussion_r976458080

Reply:
Ah, it's not needed! I don't know why I thought it was. Removed anyway in favour of using examples on the docs.